### PR TITLE
Added Arabic translation and completed French translation

### PR DESCRIPTION
**Pull request by mohamedsakhri**

### CODE REVIEW

**Comment by phw** [comment]
Thanks a lot :)

Just a note that you can also update translations on Transifex at https://www.transifex.com/phwolfer/peek . But I'm fine with pull requests, also

**Comment by phw** [comment]
Just saw, you also translated on Transifex :) I will from time to time sync the translations, so no real need to have also a pull request. 

And thanks again

**Comment by mohamedsakhri** [comment]
You are welcome :)

After translating on Transfix I waited some days before making a pull request..

BTW I'm not able to run peek in another language as English. Am I missing something? Other applications run just fine whit other env LANG locales, but peek doesn't.

**Comment by phw** [comment]
> BTW I'm not able to run peek in another language as English. Am I missing something? Other applications run just fine whit other env LANG locales, but peek doesn't.

Have you installed to `/usr/local`? I noticed that I also cannot run it in any other language when I have installed it to `/usr/local` (the default for source installs), but installed to `/usr` it works just fine. I assumed this is some misconfiguration on my system, but haven't investigated any further.

**Comment by mohamedsakhri** [comment]
Yes, it's installed in `/usr/local/bin/peek`. But this is the default behaviour. I didn't change anything in the compiling & installing steps.

**Comment by phw** [comment]
Ok, looks like this was a issue with Peek not setting the locale path. I have just pushed a change https://github.com/phw/peek/commit/477fc672d9b7606854010383a835f94e0c82646c

Can you try the current master and see whether this fixes the issue for you?

**Comment by mohamedsakhri** [comment]
It looks good now. Thanks :+1:  
A peek.desktop file isn't generated in `/usr/share/applications/` though.

**Comment by phw** [comment]
The desktop file is also put into `/usr/local/share/applications/`, and that's working for me just fine. Does your system not use it?

Another question regarding the Arabic version: Does the right-to-left layout work for you? I just tried to run Peek with an Arabic locale, and AFAIK GTK should enable the RTL layout, but it doesn't trigger.

**Comment by mohamedsakhri** [comment]
In my system the desktop file isn't generated at all. I tried to uninstall and than reinstall peek, but the same result. I'm using Linux Mint 18 XFCE.

The Arabic version looks good in my system. The RTL layout is used.

peek_003

**Comment by phw** [comment]
That's looking good :)

Regarding the missing .desktop file I can look into it. Can you open an issue here and give me the output of the build commands, especially `make` and `make install`? It should show the generation and installation of `com.uploadedlobster.peek.desktop` somewhere.

**Comment by mohamedsakhri** [comment]
I followed the output of the build commands. The desktop file is generated, but in `/usr/local/share/applications/com.uploadedlobster.peek.desktop`. I expected it to be in "/usr/share/applications", that's why I was looking there and haven't found it. 

BTW you may want to sync the translation from transifix. I changed some Arabic sentences to suite the context better.

**Comment by mohamedsakhri** [comment]
I just noticed you mentioned the path of the desktop file in a previous message. I didn't pay attention to `local` in it. Sorry.

**Comment by bil-elmoussaoui** [comment]
@phw We have the same issue with transifex not being able to auto-sync with Github repository (on FeedReader) and we didn't get the change to get it into weblate. Maybe we can give it a try to get a github bot to do this work?

Shouldn't be too hard, just update the resources whenever a new commit is pushed to github. The next step is to sync the translations from transifex to github, i don't know if there's an API or a way to detect if a change was commited to auto-sync it with github , might worth some digging

**Comment by phw** [comment]
@bil-elmoussaoui I had tried using Peek with Transifex, but it didn't work as well as I wanted. I had [txgh](https://github.com/transifex/txgh) setup for a while, which does some basic syncing between Github and Transifex. But it is limited in this regard (mostly due to API limitations of Transifex). See the discussion at https://github.com/phw/peek/issues/36#issuecomment-256444715

I favor Weblate here, its synchronization with Github is superb and it is a full two way sync (meaning I can still accept translation pull requests here and get them automatically synced back to weblate). If you want I can assist FeedReader on the localization part and setup either solution.

**Comment by bil-elmoussaoui** [comment]
@phw that would be really great!!

**Comment by mohamedsakhri** [comment]
@phw I didn't want to open a new issue for this because it's just a thought that crosses my mind whenever I take a look at the commits.
**Thought:** The synchronization of the translations with Github is great, no doubt about it. But when you take a look at the commits it's hard to follow the developpement of the App (I mean the real Code) because there are many translations' commits.
Wouldn't be better to create a repo only for the translations? I think you wouldn't have to change the project's structure, but only the git repos. Maybe a reop inside a repo..

**Comment by phw** [comment]
Definitely not a separate repository. Git has no very convenient support for this and it complicates things unnecessarily. What could be an option is to sync with a separate branch and occasionally merge back. But unless I would squash the commits on merging (which I don't want to) the individual commits will still be visible.

In the end I am quite happy with the translation commits. They are not too numerous and for me translation updates are just as important as any other change to the program code itself or other components which also are not part of the "real code" like build system or documentation.
